agent/proto/agentpb/agent.proto

```diff
@@ -167,21 +167,23 @@ message TimelineEvent {
   
   // Data contains event specific data.
   oneof data {
-    ClusterRecovered clusterRecovered = 2;
+    ClusterHealthy clusterHealthy = 2;
     ClusterDegraded clusterDegraded = 3;
 
     ClusterUpgrade clusterUpgrade = 4;
 
     NodeAdded nodeAdded = 5;
     NodeRemoved nodeRemoved = 6;
     
-    NodeRecovered nodeRecovered = 7;
+    NodeHealthy nodeHealthy = 7;
     NodeDegraded nodeDegraded = 8;
     
     ProbeSucceeded probeSucceeded = 9;
     ProbeFailed probeFailed = 10;
 
-    // TODO: Additional events
+    LeaderElected leaderElected = 11;
```

Comment:
Really a nit, I would add godocs on these exported fields so they're populated for the generated structs too.